ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -15930,18 +15932,14 @@ public ColsAndTypes(String cols, String colTypes) {
     public String colTypes;
   }
 
-  public String getInvalidAutomaticRewritingMaterializationReason() {
-    return invalidAutomaticRewritingMaterializationReason;
-  }
-
-  public void setInvalidAutomaticRewritingMaterializationReason(
-      String invalidAutomaticRewritingMaterializationReason) {
-    this.invalidAutomaticRewritingMaterializationReason =
-        invalidAutomaticRewritingMaterializationReason;
+  public MaterializationValidationResult getAutomaticRewritingValidationResult() {
+    return materializationValidationResult;
   }
 
-  public boolean isValidAutomaticRewritingMaterialization() {
-    return (invalidAutomaticRewritingMaterializationReason == null);
+  public void setAutomaticRewritingValidationResult(
+      MaterializationValidationResult materializationValidationResult) {
+    this.materializationValidationResult =
+        materializationValidationResult;
```

Comment:
nit: Since we dropped "Automatic" from the result we can as well drop it from the methods since we are renaming them anyways.

Feel free to disregard or address directly during merge without waiting for a new CI run.

Reply:
Renamed. 
There were test failures in the last CI result due to a deprecated q out file. I have to rerun it anyway.